### DISCUSSION TITLE
Please help with correct version format including branch name?

### DISCUSSION THREAD

**Discussion by eptw4by** [PaulHatch/semantic-version#168] [Q&A]
There is a `version_format` field, which, by default, looks like to be set to `"${major}.${minor}.${patch}"`. 
I wanted to include branch name and build number in tag (like `1.0.0+main.123`), but could not convince tool that those tags where created by it

**Comment by PaulHatch** [comment]
The `version_format` parameter is used only for formatting the output. Prerelease tags are not considered for the version--they should match the version anyway, because only a full release would increment to the next version.
